src/Fixer/Basic/BracesFixer.php

```diff
@@ -66,7 +66,9 @@ public function configure(array $configuration = null)
 
             if ('allow_single_line_closure' === $functionName && !is_bool($replacement)) {
                 throw new InvalidFixerConfigurationException($this->getName(), sprintf('Expected bool got "%s".', is_object($replacement) ? get_class($replacement) : gettype($replacement)));
-            } elseif ('classy_constructs' === $functionName && !in_array($replacement, $this->supportedBracePositions, true)) {
+            }
+
+            if ('classy_constructs' === $functionName && !in_array($replacement, $this->supportedBracePositions, true)) {
```

Comment:
For some weird reason keys was named wrongly.
As you are touching the could, could you rename `functionName` and `replacement` vars please ?